src/components/Player/Opsheet/Opsheet.js

```diff
@@ -8,30 +8,11 @@ class Opsheet extends Component {
     listClassName: 'list-wrapper-collapsed',
   };
 
-  checkIfCollapsed = () => {
-    if (this.state.collapsed === true) {
-      return true
-    } else {
-      return false
-    }
-  }
-
-  listClassNameChange = () => {
-    if (this.checkIfCollapsed() === true) {
-      this.setState({listClassName: 'list-wrapper'})
-    } else {
-      this.setState({listClassName: 'list-wrapper-collapsed'})
-    }
-}
+  listClassNameChange = () => { return (this.state.collapsed ? this.setState({listClassName: 'list-wrapper'}) : this.setState({listClassName: 'list-wrapper-collapsed'})) };
```

Comment:
It's a little nitpicky, but I think this is a good opportunity to point out some ways you can improve readability and code quality at the same time.

I've written my share of one-line control statements and ternary operators, so I offer this as food for thought, as opposed to criticism.

Consider this alternative:
```JavaScript
const EXPANDED_LIST_CLASSNAME = 'list-wrapper';
const COLLAPSED_LIST_CLASSNAME = 'list-wrapper-collapsed';

const changeListClassName = () => {
    const {collapsed} = this.state;
    const listClassName = (collapsed) ? EXPANDED_LIST_CLASSNAME : COLLAPSED_LIST_CLASSNAME;
    this.setState({listClassName});
};
```

A few things to call out:

1. I created two new constants which I expect would live above the class definition, at the top of the file. If you ever need to change the CSS class names for one reason or another, or you ever need to lookup what CSS classes are being used in your component, you'll thank me later. Additionally, it's best to avoid hardcoding strings inside your code. If a string really should be some constant value, consider using a properly defined constant, near the top of the file. This has the added benefit of acting as documentation for what the value is intended to represent, and how it might be used.
2. Consider naming functions in the format `<verb><noun><noun>`. For example as to why, ask yourself "What does this function do?" then read my suggested alternative: "changeListClassName".
3. There's actually quite a bit of redundancy in your code. You actually have this string, twice: `this.setState({listClassName: `.
4. If you really want to minimize the code in the file, it would seem you could use the `this.state.collapsed` value to determine if you want `list-wrapper` or `list-wrapper-collapsed`, rather than a whole function dedicated just to adding a new value to the state, based on a value that's already on the state.
5. Technically, your code returns the results of a `this.setState()` call, but I don't think `this.setState()` has a return value.

Reply:
I'll point out that using `this.setState({listClassName})` to take advantage of the shorthand notation for: `this.setState({"listClassName": listClassName})` is a convention I've become comfortable with, but technically doesn't provide maximum readability. The latter, expanded example is likely preferable to some people.